pkg/tenant/api/v1/client.go

```diff
@@ -716,7 +704,7 @@ func (s *APIv1) ProjectAPIKeyList(ctx context.Context, id string, in *PageQuery)
 
 	// Make the HTTP request
 	var req *http.Request
-	if req, err = s.NewRequest(ctx, http.MethodGet, path, nil, &params); err != nil {
+	if req, err = s.NewRequest(ctx, http.MethodGet, path, in, &params); err != nil {
```

Comment:
This is a `GET` request so the request values are passed in via `&params` which is converted into query params on lines 700-703. Sending a body (which is the `nil`/`in` parameter) is not part of the `GET` semantic.

Reply:
Thanks for catching this!